blocks/header/header.css

```diff
@@ -594,6 +527,177 @@ header .nav-bottom .other-items .mobile-lang-picker .mobile-lang-picker-item a {
   letter-spacing: -0.4px;
 }
 
+/* stylelint-disable-next-line no-descending-specificity */
+header .drop > a::after {
+  content: "";
+  height: 15px;
+  width: 12px;
+  display: inline-flex;
+  background-image: url("../../icons/arrow-down-header.svg");
+  background-size: contain;
+  background-position: top 5px center;
+  background-repeat: no-repeat;
+  border: none;
+  vertical-align: bottom;
+  margin-left: 8px;
+  position: unset;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+header ul li {
+  list-style-type: none;
+}
+
+header .nav-bottom > ul > li .mega-dropdown {
+  opacity: 0;
+  z-index: -100;
+  animation-duration: 300ms;
+  transition: all ease-in-out;
+}
+
+.mega {
+  position: absolute;
+  display: none;
+  background-color: #00587C;
```

Comment:
You could use `var(--primary)` instead...

Reply:
can we wrap this in header because its not good to straightforward use .mega here
And kindly use rem where ever possible